### ISSUE TITLE
Task Runner Explorer Bindings Stored in Separate File

### ISSUE THREAD

**Issue by someyoungideas** [madskristensen/ProjectTaskRunner#6]
Would it be possible to save the visual studio bindings in a separate file to allow for each developer to customize? I was looking for a way to use a file for the visual studio bindings that could be added to `.gitignore`, and allowing the team to not overwrite each others scripts file.

**Comment by madskristensen** [comment]
Yes, that would be possible to do. I've got other extensions that does exactly that. However, I'm not investing any more in this extension before more details about what is going to happen to project.json is being shared by the ASP.NET team. See this https://blogs.msdn.microsoft.com/dotnet/2016/05/06/net-core-rc2-improvements-schedule-and-roadmap/
